test/info.py

```diff
@@ -62,6 +62,14 @@
     'lpc11u35_archlink_if': 0x9013,
 }
 
+# Maps a target application filename (sans extension) to the ID of the board 
```

Comment:
No. sans=without. That map has as its key the file name of the target
application minus the extension.

On Wed, Apr 13, 2016 at 8:36 AM, Russ Butler <notifications@github.com>
wrote:

> In test/info.py
> <https://github.com/mbedmicro/DAPLink/pull/89#discussion_r59548020>:
>
> > @@ -62,6 +62,14 @@
> >      'lpc11u35_archlink_if': 0x9013,
> >  }
> >
> > +# Maps a target application filename (sans extension) to the ID of the board
>
> Is 'sans' a typo?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/mbedmicro/DAPLink/pull/89/files/60d34809b9a349aaca256f99756d8f475313fede#r59548020>
>